### PR TITLE
 Update: conditionally render classification

### PR DESCRIPTION
**Pull request by DanDeMicco**

### CODE REVIEW

**Comment by boxcla** [comment]
Verified that @DanDeMicco has signed the CLA. Thanks for the pull request!

**Comment by boxcla** [comment]
# Webpack Bundle Size Github Bot

| Bundle        | Diff | Percentage Diff | Before | After |
| ------------- |:----:|:---------------:|:------:|:-----:|
| Total | 0 B | 0.00% | 0 B | 0 B |

Compared against previously merged PR #438

**Powered by [webpack-bundle-size-github-bot](https://github.com/wenboyu2/webpack-bundle-size-github-bot/)**

File: src/components/ContentSidebar/SidebarFileProperties.js [hunk 1]
```diff
@@ -25,6 +25,21 @@ type Props = {
     intl: any
 };
 
+/**
+ * Gets the openModal prop for ItemProperties
+ *
+ * @param {Object} file the box file
+ * @param {Function} onClassificationClick the optional callback
+ */
+export const getClassificationModal = (file: BoxItem, onClassificationClick: ?Function) => {
```
Comment (jeremypress) [inline src/components/ContentSidebar/SidebarFileProperties.js]: Add a returns docstring

File: src/components/ContentSidebar/__tests__/SidebarFileProperties-test.js [hunk 2]
```diff
@@ -65,52 +71,86 @@ describe('components/ContentSidebar/SidebarFileProperties', () => {
...
-        const wrapper = getMountWrapper(retentionPolicyProps);
-        expect(wrapper).toMatchSnapshot();
+    describe('getClassificationModal()', () => {
+        test('should have onClassificationClick prop on SidebarFileProperties', () => {
+            expect(
+                getClassificationModal(classificationProps.file, classificationProps.onClassificationClick)
+            ).toBeUndefined();
+        });
+
+        test('should not have onClassificationClick prop on SidebarFileProperties', () => {
```
Comment (jeremypress) [inline src/components/ContentSidebar/__tests__/SidebarFileProperties-test.js]: Is the name of this test still relevant?

**Pull request merged**